### DISCUSSION TITLE
List all repository events return wrong result

### DISCUSSION THREAD

**Discussion by brisote** [community/community#54159] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hi, 

I'm using the following endpoint to fetch events for my repository:  `https://api.github.com/repos/<my_organizations_name>/<my_repo_name>/events`
([docs](https://docs.github.com/en/rest/activity/events?apiVersion=2022-11-28#list-repository-events))

The docs say that I should get "Only events created within the past 90 days will be included in timelines" but for some repositories, I get only the last 7 of history (yes I have a lot more than 90 days of history). When I open my GitHub account I can see that I have activity for this project on date 13.04 (for example) but the API return results only until 20.04 (again for example)

I'm using the API with Access Token Classic if this matters.

**Comment by brisote** [comment]
After more investigation, I found out that this problem occurs only for some of the repositories in my organization. So for some repositories, I get a full 90 days history for others I get 7 days. Is there some repository setting that can be the cause for this or it is a bug in the GitHub REST API ?

**Reply by vibin** [reply]
See: https://stackoverflow.com/q/36332950/1852422

There seems to be a limit of 300 events
